backend/BackendMIIT/src/main/java/com/example/BackendMIIT/service/EducationPlanService.java

```diff
@@ -1,6 +1,10 @@
 package com.example.BackendMIIT.service;
 
+import com.example.BackendMIIT.model.dto.EducationPlanDto;
+
 public interface EducationPlanService {
 
     void parseAndSaveEducationPlans();
+
+    EducationPlanDto getPlansByProfile(String name);

```

Comment:
Recommend to rename `getPlansByProfile` to `getPlansByProfileName`

Reply:
Renamed